### ISSUE TITLE
`wasp deps` command doesn't output the SDK deps

### ISSUE THREAD

**Issue by infomiho** [wasp-lang/wasp#2227]
We should decided if we want to keep this command, but if we do, we should make sure it outputs all the relevant deps. It now outputs the deps and dev deps used in the client and the server, but not the SDK.

**Comment by neehhaa06** [comment]
Hello @infomiho 

> The **wasp deps** command currently omits **SDK dependencies**, only listing client and server dependencies. 
To resolve this, the command’s implementation should be reviewed and updated to include **SDK dependencies,** ensuring it provides a complete view of all relevant dependencies. Testing and documenting these changes will help maintain clarity and accuracy.

**Comment by Martinsos** [comment]
Good catch! I believe the main motivation was to show these because users cared about them since they can come in conflict with their own deps. So, that is why they care about "framework code" deps. But for SDK code, they don't really need to know about the deps, as they don't have any contact with them. So it might be best if we stick to what we have, which is not showing them?
